packages/server/test/manifest-builder.test.ts

```diff
@@ -85,8 +87,14 @@ describe('manifest builder', () => {
     });
     it('updates the sourcemapURL of dist manifest with fingerprinted file', () => {
       expect(distMapURL).toMatch(/manifest-[0-9a-f]+\.js.map/);
-    })
-  })
+    });
+    it('warning when sourcemapURL is not generated at the bottom', async () => {
+      // wip
+      await writeFile('tmp/manifest-builder/fakeManifest.js', 'hello');
+      let error = await updateSourceMapURL('tmp/manifest-builder/fakeManifest.js', '');
```

Comment:
I think it's fine to have a fixture for this that is always there, lke `test/fixtures/manifest-without-source-mapping.js`

Also, in JavaScript we should use normal error handling rather than use error codes. In other words `updateSoureMapURL` should raise an exception.

Reply:
I'll just use the current `test/fixtures/empty.js`.

How can I test for an `Error`? I'm having difficulty getting an Operation to return anything but an empty object.

Reply:
You have to run the operation. This is what the `actions.fork` helper is for. It will convert the operation into a promise that resolves to the result of running the operation, or rejects if an operation fails.

```ts
expect(actions.fork(updateSourceMapURL('test/manifest/empty.js').rejects.toMatch({message: "blah blah"});
```